alfresco-transformer-base/src/main/java/org/alfresco/transformer/util/MimetypeMap.java

```diff
@@ -178,4 +178,10 @@
     String MIMETYPE_VORBIS = "audio/vorbis";
     String MIMETYPE_FLAC = "audio/x-flac";
     String MIMETYPE_ACP = "application/acp";
+    String MIMETYPE_STC = "application/vnd.sun.xml.calc.template";
```

Comment:
Has anyone noticed that there is a very similar file in the model? https://github.com/Alfresco/alfresco-transform-model/pull/62/files

Reply:
@eknizat Please raise an improvement idea to potentially rationalise in the future, to see whether it is feasible to de-dupe ? However, since part of T-Base, we need to consider backwards-compatibility for custom T-Engines / custom tests ?

Reply:
is it OK to merge despite this?

Reply:
Yes